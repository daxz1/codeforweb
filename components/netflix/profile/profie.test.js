/* global describe, it, expect */
import React from 'react';
import renderer from 'react-test-renderer';
import Profile from './profile';

describe('[RENDER] components/netflix/profile/profile.js', () => {
  it('Header Component renders', () => {
    const component = renderer.create(<Profile name="Morwen" picture="https://occ-0-299-300.1.nflxso.net/art/c83fe/6c218aba13637fff1554e17c73c4e2481e6c83fe.png" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
