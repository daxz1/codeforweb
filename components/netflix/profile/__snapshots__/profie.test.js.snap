// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[RENDER] components/netflix/profile/profile.js Header Component renders 1`] = `
<article
  className="profiles__profile"
  style={
    Object {
      "backgroundImage": "url(https://occ-0-299-300.1.nflxso.net/art/c83fe/6c218aba13637fff1554e17c73c4e2481e6c83fe.png)",
    }
  }
>
  <div
    className="profile__title"
  >
    Morwen
  </div>
</article>
`;
